.github/workflows/build-images.yaml

```diff
@@ -30,14 +30,6 @@ jobs:
         id: add-branch-tag
         run: |
           echo "IMG_TAGS=${GITHUB_REF_NAME/\//-} ${{ env.IMG_TAGS }}" >> $GITHUB_ENV
-      - name: Set Authorino version
-        id: authorino-version
-        run: |
-          if [[ ${GITHUB_REF_NAME/\//-} =~ ^v[0-9]+\.[0-9]+\.[0-9]+(-.+)?$ ]]; then
-          echo "VERSION=${GITHUB_REF_NAME/\//-}" >> $GITHUB_ENV
```

Comment:
So users who run `./authorino version` will no longer see a semantic version not even on official releases? Not sure if I like this.

What about aiming for something more structured? E.g.

```json
{"version":"latest","commit":"abcd1234","dirty":true}
```

```json
{"version":"v0.17.0","commit":"efgh4567","dirty":false}
```

Reply:
That makes total sense, Gui. I will aim to resolve this.